index.html

```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
+            I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
           <button class="blackButton">
              Start collaboration
           </button>
```

Comment:
- [ ] This section does not match the Figma design. Notice the point marked with the red circle. The button there should be within the dark-blue background. Not just the button, please ensure the correct margin, position and layout are the same with the Figma design. Please compare this image below 👇🏾 

Your design
![image](https://user-images.githubusercontent.com/84156714/221373078-b9009d3b-92a4-4635-bcb0-e51932eefe43.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/221373127-8eb65176-c25c-4530-b408-591f33924113.png)

Reply:
Fixed.